README.md

```diff
@@ -1,5 +1,28 @@
 # FCC Test Suite Template
 
-**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+> Welcome to my first microverse Project.
 
-You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
+## Getting Started
+
```

Comment:
- [ ] Please you are required to act as a code reviewer and check this PR for issues concerning HTML and CSS best practices and the use of HTML semantic tags. Kindly go through the files changed and give your review.

- [ ] Please use the inline comments GitHub feature to give at least two comments.

- [ ] Please always add an explanation, a "why", for each requested change. For example, this is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
This request lacks a "why": *Why don't you use an article tag here instead of a div?*

- [Optional] Consider using a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.🙃😁